scheduler-workflow/workflow.go

```diff
@@ -0,0 +1,225 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	http2 "net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	dapr "github.com/dapr/go-sdk/client"
+	"github.com/dapr/go-sdk/service/http"
+	"github.com/dapr/go-sdk/workflow"
+)
+
+const appPort = ":8484"
+
+var stage int
```

Comment:
This is probably never going to be accessed in parallel, but just to keep it clean, since it's a global variable and can be accessed from different goroutines, let's make it an atomic int.